src/components/VmUserMessages/index.js

```diff
@@ -69,12 +65,10 @@ class VmUserMessages extends React.Component {
 
     return (
       <li className='dropdown'>
-        <a href='#' onClick={hrefWithoutHistory(onToggle)}>
-          <div className={isUnread(userMessages) ? style['usermsgs-unread'] : style['usermsgs-allread']}>
-            <span className='pficon pficon-info' />&nbsp;{msg.messages()}
-          </div>
+        <a className='dropdown-toggle nav-item-iconic' href='javascript:;' title={msg.messages()} onClick={hrefWithoutHistory(onToggle)}>
```

Comment:
Please replace value of `href` with [idiomatic][1] `#` or leave the attribute empty. `javascript:;` in not necessary in combination with `hrefWithoutHistory()` function.

[1]: https://reactjs.org/docs/handling-events.html